src/tests/NoVoidZeroRuleTests.ts

```diff
@@ -0,0 +1,62 @@
+import { Utils } from '../utils/Utils';
```

Comment:
Would you mind switching this to the newer test format, actually? Under the `test/` directory, copy any one that doesn't have sub-directories _(which would be necessary if this rule has options)_ such as `non-literal-fs-path`, modify the `tslint.json` to only include `no-void-0`, and add your test cases there. 

Tests in general are moving to that format (#489) because it's quite a bit nicer to read & write.

Reply:
Sure! Good to know -- thank you!